roles/designer.md

```diff
@@ -1,25 +1,27 @@
 # Designer 
 
-- [SFIA: Level 3](https://github.com/madetech/handbook/blob/main/roles/sfia/designer.md)
 - Salary outside London: £36,000 - £45,000 
 - Salary inside London: £40,000 - £50,000
+- Location: Birmingham, Bristol, Glasgow, London, Manchester or Newcastle
+- Hybrid working: Client-site/office 1-2 days per week
+- SFIA: [Level 3](sfia/designer.md)
 
-Designers might aleady be a product designer, UX designer, service designer or interaction designer. 
+Designers might already be a product designer, UX designer, service designer or interaction designer. 
 
 ## Summary
 
-Made Tech wants to positively impact the future of the country by using technology to improve society. We believe being design-led can create positive outcomes in the public sector, through critical services enabled by technology. We are building a community of designers and researchers to support the public sector growing demand for a design-led approach to service delivery. 
+Made Tech wants to positively impact the country's future by using technology to improve society. We believe being design-led can create positive outcomes in the public sector through critical services enabled by technology. We are building a community of designers and researchers to support the public sector growing demand for a design-led approach to service delivery.
 
-Designers are experienced practitioners who collaborate with others to tackle challenges faced by people and society. They work within a team to design and deliver public services. They do this by influencing the direction, assuring the quality of design delivery within their team, and by leading the end-to-end design of a chosen service. They are active members of a healthy User-Centred Design (UCD) community and culture at Made Tech. 
+Designers are experienced practitioners who collaborate with others to tackle challenges faced by people and society. They work within a team to design and deliver public services. They do this by influencing the direction, assuring the quality of design delivery within their team and by leading the end-to-end design of a chosen service. They are active members of a healthy User-Centred Design (UCD) community and culture at Made Tech. 
 
 ## Key responsibilities 
 
-- Responsible to the Lead Designer for success of designing and delivering good public services
-- Turn team conversations and research into drawings and prototypes, to create a shared understanding of problems to solve and ideas to test
-- Make things real, by caring about content and design enough to ensure everyone can use public services
-- Make public services simple to explain and understand, by removing complexity and keeping designs focused on creating outcomes for people and society
+- Responsible to the Lead Designer for successfully designing and delivering of good public services
```

Comment:
Can you be "responsible to" someone? I think "responsible for" or "reporting to" maybe?